doc/getting-started/getting-started.rst

```diff
@@ -322,9 +338,15 @@ Generate a Board Configuration File
          disk="/media/$USER/"$(ls /media/$USER)
          cp -r $disk/board_inspector ~/acrn-work
 
-#. On the target, run ``board_inspector.py`` (the board inspector tool) to generate
-   the board configuration file. This example uses the parameter ``my_board``
-   as the file name.
+#. On the target, install a dependency (``msr``) and run ``board_inspector.py``
+   (the board inspector tool) to generate the board configuration file. This
+   example uses the parameter ``my_board`` as the file name.
+
+   .. code-block:: bash
+
+      sudo modprobe msr
+
+   Followed by:
```

Comment:
Recommend formatting these steps as substeps "a" and "b" to make it clear that you're running the tool, rather than using the wording "followed by"